dockerfile/build_image.sh

```diff
@@ -96,8 +96,8 @@ if [[ "$build_type" == "cuda" ]]; then
     popd
 elif [[ "$build_type" == "rocm" ]]; then
     echo "AMD runtime selected"
-    echo not supported yet but coming soon
-    exit 1
+    IMAGE="azurenodehealthchecks.azurecr.io/staging/aznhc/aznhc-rocm"
```

Comment:
Will we have to pull this image as well along with "mcr.microsoft.com/aznhc/aznhc-nv"?

Reply:
Maybe. Not a high priority at the moment